docs/proposals/002-api-proposal/glossary.md

```diff
@@ -0,0 +1,94 @@
+    -   [Latency Based Routing](#latency-based-routing)
+    -   [Lora Affinity](#lora-affinity)
+
+
+<!-- /toc -->
+
+## API
+This is a very brief description of terms used to describe API objects, included for completeness.
+
+### BackendPool
+A grouping of model servers that serve the same set of fine-tunes (LoRA as a primary example). 
+
+Shortened to: `BEP`
+
+### UseCase
+An LLM workload that is defined and runs on a BackendPool with other use cases.
+
+# Capacity Constrained Routing
+
+## Priority
+
+### Summary
+Priority specifies the importance of a UseCase relative to other usecases within a BackendPool. 
+
+### Description
+
+For our purposes, priority can be thought of in two classes:
+- Critical
+- Non-Critical
+
+The primary difference is that non-critical UseCase requests will be rejected in favor of Critical UseCases the face of resource scarcity. 
+
+Example: 
+
+Your current request load is using 80 Arbitrary Compute Units(ACU) of your pools total of 100ACU capacity. 40ACU are critical workload requests, 45 are non-critical. If you were to lose 30 ACU due to an unforseen outage. Priority would dictate that of the 10 surplus ACU to be rejected the entirety of them would be from the non-critical requests. 
+
+## Fairness
+
+### Summary
+Fairness specifies how resources are shared among different UseCases, in a way that is most acceptable to the user.
+
+### Description
+
+Fairness, like priority, is only used in resource scarcity events. 
+
+Fairness is utilized when requests of the same priority class need to be rejected, or queued. There are many dimensions that could be considered when considering shared resources. To name a few:
+- KV-cache utilization
+- Total request count
+- SLO adherence
+
+For the v1 MVP, the only objective a User can specify is the SLO objective they would like to meet. So, in following that pattern, fairness in MVP will simply be considered for SLO adherence. SLO Adherence is only being considered over a rolling time window of data. 
+
+The TTL we are currently assuming is: `5 min` 
+
+### Example
+
+**Assumption:** Services have equally weighted fairness for this example.
+
+- Service A has been meeting its SLO 98% of the requests made in the time window, and Service B has met the SLO 94% of the time.
+
+- A request for both Service A and Service B come in at the same time, and there is only capacity to start a single new request in the BEP, this capacity would meet the SLO for both services. The other request would be queued (potentially causing that request to not meet SLO).
+
+- To fairly share these resources. Service B *must* be selected to begin the request immediately as Service A has had its SLO met a larger percentage of the time.
+
+# General Routing
+Different from the previous definitons, these terms are used to describe methods of routing that are constant, and seek to better utilize compute resources to avoid capacity constraints as much as possible.
+
+## Latency Based Routing
```

Comment:
I think latency based routing has to do with priority. As far as the concerned useCases here are the one with an Objective (Critical). It's more likely how under the hood we prioritize critical useCases within a BackendPool: which useCase should I route in priority to the best available Backend.
Let me know what you think.

Reply:
It does, I try to describe Priority up above. This was meant to just describe what Latency based routing means when we reference it, to help explain the SLO field. On the SLO field we mention that priority is implicitly added. 

LMK if that suffices or if you feel we should go into further detail here. Thanks!